test/domain_support_spec.js

```diff
@@ -1,57 +1,61 @@
-
-var domain = require("domain");
-var _ = require("underscore");
-
-describe("errormailer domain support", function() {
-
-  var transport;
-  var transportSpy;
-  var afterSend;
-  var d;
-  var uncaughtExceptionHandler;
-
-  function verifyMailWithOpts(opts) {
-    expect(transportSpy).to.have.been.calledWith(sinon.match(opts));
+const domain = require('domain')
```

Comment:
I think we can remove domain support altogether.

Reply:
you sure? isn't that decreasing test coverage?